data/compatibility/ossmc.yaml

```diff
@@ -1,9 +1,12 @@
 # The Compatible Version Matrix between OSSMC, OCP and Kiali
 # See: content/en/docs/Installation/installation-guide/prerequisites.md -> {{<compat-table-ossmc>}}
 versionRange:
+  - ocpVersion: "4.15+"
+    minOSSMCVersion: "1.84"
```

Comment:
Is this supposed to be 1.74?

Reply:
No, the migration to PF5 was done for OSSMC 1.84 release. Actually, OSSMC 1.73 to 1.83 contains the Kiali code from 1.73 (version release was done to align versions).